### ISSUE TITLE
Set Keyboard Focus and Window Focus

### ISSUE THREAD

**Issue by bobort** [Aldaviva/AuthenticatorChooser#29]
I use a PIN to use my security device. The Windows Security dialog does pop up, but it stays behind all the windows typically. It also doesn't have the keyboard focus so that I can just start typing the PIN. It's a convenience issue, I know.

**Comment by Aldaviva** [comment]
@bobort @AFulgens what operating system versions are you experiencing this issue on? I'm using Windows 11 24H2 (build 26100.4770) and am having trouble reproducing it now. I only remember this bug from Windows 10. Thanks.

**Comment by AFulgens** [comment]
`winver` says Version 23H2 (OS Build 22631.5624). Also note that sometimes randomly AuthenticationChooser does not trigger, but next time around it will. I could not pinpoint yet under what circumstances it won't trigger, even when the pop-up has the focus (I'd say it's less than 5% of cases in my experience), thus I did not open a ticket yet.

**Comment by bobort** [comment]
Windows 11 Version 23H2 (OS Build 22631.5472)

<img width="344" height="308" alt="Image" src="https://github.com/user-attachments/assets/e94d4a06-5606-4715-83bb-2bca378ff7bd" />

**Comment by felipecrs** [comment]
I think this project helps with focusing the window:

- https://github.com/FroggMaster/AuthAlwaysOnTop
